### PR TITLE
TVH update

### PR DESCRIPTION
**Pull request by th0ma7**
_Motivation:_  Update DTV tables and to latest git hash
_Linked issues:_  #4680 and #4545

### Checklist
- [ ] Build rule `all-supported` completed successfully
- [ ] Package upgrade completed successfully
- [ ] New installation of package completed successfully

### CODE REVIEW

**Comment by th0ma7** [comment]
@hgy59 with the recent DSM7 wizard changes... I guess I should ensure to circumvent this somehow?

**Comment by th0ma7** [comment]
Ready for review.
Package build is looking quite good and been performing well for my own personal usage.
Currently available for beta testing through my github.
I intend to bump package version when ready for publishing (along with ffmpeg and others).

File: cross/dtv-scan-tables/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = nop
+INSTALL_TARGET = dtv_scan_tables_install
+
+HOMEPAGE = https://github.com/crazycat69/dtv-scan-tables
+COMMENT  = Digital TV scan tables
+LICENSE  = N/A
+
+include ../../mk/spksrc.cross-cc.mk
```
Comment (hgy59) [inline cross/dtv-scan-tables/Makefile:19]: you can omit `CONFIGURE_TARGET = nop` and `COMPILE_TARGET = nop` when you use 
```suggestion
include ../../mk/spksrc.install-resources.mk
```

**Review by hgy59 — CHANGES_REQUESTED** [review]
two minor suggestions

**Comment by th0ma7** [comment]
> two minor suggestions

@hgy59 thnx, applied!

**Pull request merged**